recipes/texlive-core/build.sh

```diff
@@ -0,0 +1,52 @@
+#! /bin/bash
+
+export PKG_CONFIG_LIBDIR="$PREFIX/lib/pkgconfig:$PREFIX/share/pkgconfig"
+
+mkdir -p tmp_build && pushd tmp_build
+  ../configure --prefix=$PREFIX \
+               --with-cairo=$PREFIX \
+               --datarootdir="$PREFIX/share/texlive" \
+               --disable-all-pkgs \
+               --disable-native-texlive-build \
+               --disable-ipc \
+               --disable-debug \
+               --disable-dependency-tracking \
+               --disable-mf \
+               --disable-pmp \
+               --disable-upmp \
+               --disable-aleph \
+               --disable-eptex \
+               --disable-euptex \
+               --disable-luatex \
+               --disable-luajittex \
+               --disable-uptex \
+               --enable-web2c \
+               --enable-silent-rules \
+               --enable-tex \
+               --enable-etex \
+               --enable-pdftex \
+               --enable-xetex \
+               --enable-web-progs \
+               --enable-texlive \
+               --enable-dvipdfm-x \
+               --with-icu=$PREFIX \
+               --with-mpfr=$PREFIX \
+               --with-gmp=$PREFIX \
+               --with-pixman=$PREFIX \
+               --with-freetype2=$PREFIX \
+               --with-libpng=$PREFIX \
+               --with-zlib=$PREFIX \
+               --x-libraries=$PREFIX/lib \
+               --x-includes=$PREFIX/include \
```

Comment:
While that sounds crazy it isn't entirely, see this [recipe]( https://github.com/ostrokach/conda-recipes-extra/tree/8ea3eab3de34f1ee043a7df3c4a24e44b9e48e18/xorg ).